public/stylesheets/main.css

```diff
@@ -0,0 +1,5 @@
+* {
+  background-color: purple;
```

Comment:
Do you know why the outputted file is missing the `color: white;` property?

Reply:
I think it's because i added the white, but i did not run the compiler